content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,205 @@
+
+This release includes a total of 37 enhancements: eleven of them are graduating to Stable, ten are graduating to Beta, and sixteen of them are entering Alpha. We also have two features being deprecated or removed.
+
+## Release theme and logo 
+
+**Kubernetes 1.26: Electrifying**
+
+\<image here>
+
+The theme for Kubernetes v1.26 is _Electrifying_.
+
+Each Kubernetes release is the result of the coordinated effort of dedicated volunteers, and only made possible due to the use of a diverse and complex set of computing resources, spread out through multiple datacenters and regions worldwide. The end result of a release - the binaries, the image containers, the documentation - are then deployed on a growing number of personal, on-premises, and cloud computing resources. 
+
+In this release we want to recognise the importance of all these building blocks on which Kubernetes is developed and used, while at the same time raising awareness on the importance of taking the energy consumption footprint into account: environmental sustainability is an inescapable concern of creators and users of any software solution, and the environmental footprint of sofware, like Kubernetes, an area which we believe will play a significant role in future releases.
+
+As a community, we always work to make each new release process better than before (in this release, we have [started to use Projects for tracking enhancements](https://github.com/orgs/kubernetes/projects/98/views/1), for example). If v1.24 "Stargazer" [had us looking upwards, to what is possible when our community comes together](/blog/2022/05/03/kubernetes-1-24-release-announcement/), and v1.25 "Combiner" [what the combined efforts of our community are capable of](/blog/2022/08/23/kubernetes-v1-25-release/), this v1.26 "Electrifying" is also dedicated to all of those whose individual motion, integrated into the release flow, made all of this possible.
+
+## Major Themes
+    
+Kubernetes v1.26 is composed of many changes, brought to you by a worldwide team of volunteers. For this release, we have identified several major themes.
+
+### Change in container image registry
+
+In the previous release, [Kubernetes changed the container registry](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/), allowing the spread of the load across multiple Cloud Providers and Regions, a change that reduced the reliance on a single entity and provided a faster download experience for a large number of users.
+
+This release will be the first one that will be exclusively published in the new [registry.k8s.io](http://registry.k8s.io) image registry: no tags from v1.26 will be published, and only tags from releases before v1.25 will continue to be updated, in the (now legacy) k8s.gcr.io image registry. Check [this recent article](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) for more information on the motivation, advantages, and implications of this significant change.
```

Comment:
```suggestion
This release will be the first one that will be exclusively published in the new [registry.k8s.io](http://registry.k8s.io) image registry: no tags from v1.26 will be published, and only tags from releases before v1.25 will continue to be updated, in the legacy k8s.gcr.io container image registry.
```

remove the last sentence since we already mentioned the article the sentence before?

Reply:
Yes, we should avoid the repetition. OTOH, I think it's important to point people towards the explanation. What I will do here is to replace the first link with the initial registry PR that gost merge in v1.25, and then keep the last sentence as an additional source of information. THis, I think works well - it's the first link that is out of context IMO.